src/main/java/com/project/momo/common/exception/ErrorCode.java

```diff
@@ -31,7 +32,11 @@ public enum ErrorCode {
 
     //ClubService
     CLUB_NOT_FOUND(HttpStatus.NOT_FOUND, "존재하지 않는 모임입니다."),
-    DUPLICATED_CLUB_NAME(HttpStatus.CONFLICT, "이미 존재하는 모임 이름입니다.");
+    DUPLICATED_CLUB_NAME(HttpStatus.CONFLICT, "이미 존재하는 모임 이름입니다."),
+    EXCEED_CLUB_SIZE_LIMIT(HttpStatus.BAD_REQUEST, "최대 가입 인원을 초과하였습니다."),
+    DUPLICATED_CLUB_JOIN(HttpStatus.CONFLICT, "이미 가입한 모임입니다."),
+    EXCEED_CLUB_CREATION_LIMIT_PER_MEMBER(HttpStatus.BAD_REQUEST, "최대 모임 생성 개수를 초과하였습니다."),
+    CONSIST_NOT_FOUND(HttpStatus.NOT_FOUND, "해당 클럽에 가입되어 있지 않습니다.");
```

Comment:
한글로는 모임, 영문으로는 club
용어를 통일해주는게 좋습니다
```suggestion
    CONSIST_NOT_FOUND(HttpStatus.NOT_FOUND, "해당 모임에 가입되어 있지 않습니다.");
```

Reply:
넵, 헷갈리지 않도록 통일하고 주의하겠습니다.